doc/ale.txt

```diff
@@ -216,6 +216,18 @@ g:ale_echo_cursor                                           *g:ale_echo_cursor*
   error at a column nearest to the cursor when the cursor is resting on a line
   which contains a warning or error. This option can be set to `0` to disable
   this behaviour.
+  The format of the message can be customizable in |g:ale_echo_msg_format|.
+
+
+g:ale_echo_msg_format                                    *g:ale_echo_msg_format*
+
+  Type: |String|
+  Default: `%s`
+
+  This variable defines the format of the echoed message. The `%s` is the
+  error message itself, and it can contain the following handlers:
+  - `%linter%` for linter's name
+  - `%severity%` for the type of severity
```

Comment:
Maybe. One thing I found really helpful when trying out vim-airline was that I could type `:help some_variable`, and the section would probably have some good suggestions on how to set the variables up. That was how I figured out how to get the triangular arrows to work.